src/styles/main.scss

```diff
@@ -0,0 +1,60 @@
+@import './utils/variables';
+
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  padding: 10px;
+  flex-wrap: wrap;
+  gap: $base-gap;
+  width: ($base-size * 7) + ($base-gap * 6);
```

Comment:
The calculation of the width for the `.calendar` might not include the padding. According to the box-sizing property which is by default 'content-box', padding is not included in the width. If you want the padding to be included, you should set `box-sizing: border-box;` for `.calendar`.